INSTALLING.md

```diff
@@ -16,7 +16,7 @@ These instructions will guide you through the installation of Tenstorrent system
 
```

Comment:
Can we add a note at the top of the file to the effect of "This script is intended for advanced users and developers who intend to build Metalium from source"? That helps separate concerns from the quickstart.md file on the docs site.